.github/ISSUE_TEMPLATE/Question.md

```diff
@@ -0,0 +1,23 @@
+---
+name: 💬 Question
+about: If you have a question, please check out our community resources!
+
+---
+
+# Questions
+
+Issues on GitHub are intended to be related to bugs or feature requests with provider codebase,
```

Comment:
Should we allow issues to be used for questions? I mean it shouldn't be a problem now given that only we are using our modules and there isn't any good way for people to contact us.

Reply:
Would it be possible to have an open slack channel for the open-source stuff?